.circleci/config.yml

```diff
@@ -182,17 +180,7 @@ jobs:
             export TAG="${COMMIT}"
             export CONTAINER_REPO="networkservicemeshci"
             export GOPROXY=https://proxy.golang.org,direct
-            make docker-<< parameters.container >>-build
-            make docker-<< parameters.container >>-push
-
-# build ready - empty job
-  build-container-success:
-    docker:
-      - image: circleci/golang:1.13-stretch
-    steps:
-      - run:
-          command: |
-            echo "SUCCESS"
+            make -j 8 docker-push
```

Comment:
We can use just '-j' option (no concrete number) complemented by load restriction option '-l'.
Also, log aggregation option '--output-sync=line' might be a useful addition
https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_toc.html#TOC47

Reply:
The physical machine underlying has 8 cores.  I briefly experimented with -j 16, but it didn't help much.  If you have more advice on how to tune, I'd love to take it :)

As to --output-sync=line ... this is a very good idea... why line and not target though?